ui/components/ui/actionable-message/README.mdx

```diff
@@ -0,0 +1,47 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+
+import ActionableMessage from '.';
+
+# Actionable Message
+
+Popup component that give the user information. Actionable Message component can generate a tooltip and a maximum of two action buttons.
```

Comment:
I don't think the actual component has a "popup" function? It may also cause confusion if we have a popup component in future. 

```suggestion
An ActionableMessage displays an important message that attracts the user's attention. It can also generate a tooltip and provide a maximum of two action buttons.
```

Reply:
Yes good point